lib/debug/source_repository.rb

```diff
@@ -16,7 +16,7 @@ def add iseq, src
     def add_path path, src: nil
       case
       when src
-        if File.file?(path)
+        if File.file?(path.to_s)
```

Comment:
```ruby
path && File.file?(path)
```
might be better cause it doesn't need to call `File.file?` with an empty string.

Reply:
I had that originally and went back and forth trying to decide before submitting the PR, since I didn't personally have a preference.

Reply:
I'm going to modify it with the conditional to save on the method call.